discovery-provider/src/tasks/entity_manager/user.py

```diff
@@ -123,9 +124,31 @@ def create_user(params: ManageEntityParameters):
         is_current=False,
     )
 
-    if params.metadata == "v2":
+    user_metadata = None
+    try:
+        # for single tx signup
+        user_metadata = json.loads(params.metadata)["data"]
```

Comment:
this should be parsed out when the params are built in entity_manager_update https://github.com/AudiusProject/audius-protocol/blob/83240a36c2fcd59b3985fe13315ddd8e52028a6e/discovery-provider/src/tasks/entity_manager/utils.py#L159

so just use params.metadata

Reply:
actually nevermind, what you have is correct, it doesnt expect metadata and therefore doesnt attempt to parse for CREATE USER events